File: Routes/temperature.py
from flask_restplus import Namespace, Resource
from Conversions import temperature

ns_temperature = Namespace(
    'Temperature', description='Temperature calculations')


@ns_temperature.route('/Celcius/<string:input>/ToFehrenheit')
class Conversions(Resource):
    def get(self, input):
        try:
            input = float(input)
            return temperature.celcius_to_fehrenheit(input)
        except ValueError:
            return "Can't convert {} to number".format(input), 400


@ns_temperature.route('/Fehrenheit/<string:input>/ToCelcius')
class Conversions(Resource):
    def get(self, input):
        try:
            input = float(input)
            return temperature.fehrenheit_to_celcius(input)
        except ValueError:
            return "Can't convert {} to number".format(input), 400

File: routes/temperature.js
const apiBasePath = '/api/v2/Temperature/';

export function celciusToFehrenheit(input) {
  return (input * 1.8) + 32;
}

export function fehrenheitToCelcius(input) {
  return (input - 32) * 0.555555555556;
}

export const temperatureRoutes = [
  {
    method: 'GET',
    path: `${apiBasePath}celciusToFehrenheit/{input}`,
    handler: (request) => celciusToFehrenheit(request.params.input)
  },
  {
    method: 'GET',
    path: `${apiBasePath}fehrenheitToCelcius/{input}`,
    handler: (request) => fehrenheitToCelcius(request.params.input)
  }
];
